### PR TITLE
chore(statefulset) support tolerations

### PR DESCRIPTION
**Pull request by jgomez-jb**

### CODE REVIEW

**Review by jb-wau — DISMISSED** [review]
Approved.

File: charts/agent-group/values.schema.yaml [hunk 1]
```diff
@@ -108,3 +109,6 @@ properties:
   env:
     description: List of additional environment variables that may be specified in the container
     $ref: "#/definitions/env"
+  tolerations:
+    description: Pod tolerations to node tains
```
Comment (tmercswims) [inline charts/agent-group/values.schema.yaml]: Typo?

Comment (jgomez-jb) [reply charts/agent-group/values.schema.yaml]: @tmercswims I think it is correct, it should be `tolerations` or maybe I am not spotting the right word where I might have made a typo.

**Review by jb-wau — APPROVED** [review]

**Review by tmercswims — APPROVED** [review]

**Review by dzermeno-jb — APPROVED** [review]

**Comment by jb-wau** [comment]
Merging.

**Pull request merged**